neps/nep-0463.md

```diff
@@ -0,0 +1,174 @@
+```
+  NEP: 463
+  Title: Decentralized Autonomous Organization Standard
+  Author: DAO Builders Group
+  Status: Draft
+  DiscussionsTo: nearbuilders.com/tg-dao
+  Type: Standards
+  Created: 21-Feb-2023
+```
```

Comment:
Please, don't use code formatting here. Follow [the template](https://raw.githubusercontent.com/near/NEPs/master/nep-0000-template.md).